package.xml

```diff
@@ -12,7 +12,8 @@
   <license>BSD 3-Clause</license>
 
   <buildtool_depend>ament_cmake</buildtool_depend>
-
+  <buildtool_depend>cmake_always_do</buildtool_depend>
```

Comment:
Is this really a buildtool_depend and not a build_depend?  I have to admit, I'm not sure what the actual difference is...

Reply:
I could also not find any proper documentation, but I agree that intuitively build_depend sounds better